integration-tests/pyproject.toml

```
@@ -5,6 +5,7 @@ description = "Text Generation Inference integration tests"
 authors = ["Nicolas Patry <nicolas@huggingface.co>"]
 
 [tool.poetry.dependencies]
+pydantic = "> 1.10, < 3"
```

```suggestion
pydantic = "> 2, < 3"
```
no ? 2 is not compatible with 1 iirc.